index.html

```diff
@@ -0,0 +1,90 @@
+                    </li>
+                    <li>
+                        <button>Support</button>
+                    </li>
+                    <li class="right button">
+                        <i class="fa fa-search custom-user"  aria-hidden="true"></i><input type="search" name="search">
+                    </li>
+                </ul>
+            </nav>
+            <div>
+                <h1>What will your verse be?</h1>
+                <p class="sub-heading">Everyone has something to share.See how people are using
+                    <br>iPad in their unique ways to add to the world's story
+                    <br>
+                    Watch the "Your verse" film and explore the stories
+                    <i class="fa fa-play-circle-o" aria-hidden="true"></i>
+                </p>
+            </div>
+        </div>
+        <aside>
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5s_2x.jpg"  alt="iPhone 5s. Forward thinking.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_iphone5c_2x.jpg" alt="iPhone 5c. For the colorful.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_30_years_2x.jpg"  alt="Happy Birthday, Mac. Explore 30 years of innovation on the Mac.">
+            <img src="https://web.archive.org/web/20140301004610im_/http://images.apple.com/home/images/promo_supplier_responsibility_2x.jpg" alt="Supplier Responsibility The latest results of our worldwide efforts.">
+        </aside>
+        <footer class="footer">
+            <div class="first-footer">
+                <p>
+                    Shop the Apple Online Store (1-800-MY-APPLE), visit an <a href="www.apple.com">Apple Retail Store</a> , or find a <a href="www.apple.com">reseller</a>.
+                </p>
+                <ul>
+                    <li><a href="www.apple.com"> Site Map </a></li>
+                    <li><a href="www.apple.com"> Hot News </a></li>
+                    <li><a href="www.apple.com"> RSS Feeds </a></li>
+                    <li><a href="www.apple.com"> Media Info </a></li>
+                    <li><a href="www.apple.com"> Environment </a></li>
+                    <li><a href="www.apple.com"> Job Opportunities </a></li>
+                    <li><a href="www.apple.com"> Contact Us </a></li>
+                    <li>
+                        <img class= "country-icon" src="images/country.png" alt="Choose your country or region">
+                    </li>
+                </ul>
+            </div>
+            <div class ="second-footer">
+                <span>
+                    Copyright © 2014 Apple Inc. All rights reserved. 
+                    <a href="www.apple.com">Terms of Use</a>
+                    <a href="www.apple.com">Privacy Policy</a>   
+                </span>
+            </div>
```

Comment:
- [ ] This second footer should be aligned to the left with the `first-footer` and it's not. See image:
![Screenshot from 2020-03-16 17-41-43](https://user-images.githubusercontent.com/51926937/76808444-6cc3dd00-67ad-11ea-95a8-3a2deae50acc.png)